.github/workflows/ci.yml

```diff
@@ -3,9 +3,9 @@ name: Continuous Integration
 # On every pull request on the develop branch, execute all tests [CI]
 on:
   pull_request:
-    branches:
-      - develop
```

Comment:
Should we remove `develop` now that it has been deleted and we aren't using it?

Reply:
I think you have clicked on the file showing the deleted lines